examples/playground/overlays/example-append-array-item/append-array-item.yml

```diff
@@ -3,13 +3,13 @@
 #@overlay/match by=overlay.all
 ---
 awards:
-#@overlay/append
```

Comment:
i actually think this example of using overlay/append should stay as is since it's showing how to use overlay/append.

Reply:
but it could also have an additional variation that does not use overlay/append